.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,212 @@
+name: Pull Request
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  lint:
+    name: Format & Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
```

Comment:
It's nice to get the latest lint checks but I'd prefer to stay on `stable` on CI for clippy and rustfmt to reduce churn on format / clippy errors. I expect most contributors are using stable when contributing.

Thoughts?

Reply:
You're right, it wouldn't be great to surprise contributors with errors that don't occur on stable.

Reply:
Do you think it's valuable to keep as an optional job where failures don't affect mergeability?

Reply:
GitHub Actions sadly doesn't support this properly right now (see https://github.com/actions/toolkit/issues/399)...
My current settings *should* allow the workflow to succeed even if the nightly lint fails. The lint job is marked as `continue-on-error` and its documentation says this:
> Prevents a workflow run from failing when a job fails. Set to true to allow a workflow run to pass when this job fails.

However, from the looks of it, that just prevents the workflow from cancelling all the other tasks. The check is still marked as "failed" so I wouldn't be able to merge it. I can't say for certain whether that is true because the broken Travis check on this PR prevents me from merging it anyway.

The only other way I know of is to mark the individual steps as `continue-on-error` but then there would be no indication of failure whatsoever. Regardless of whether clippy reported any warnings, the check would always succeed. The warnings are visible in the annotations (like here: https://github.com/yewstack/yew/actions/runs/209847977) but I don't think that's particularly helpful...

Perhaps it would be best to only keep run the lints on stable for now?

Reply:
That's too bad, yeah just stable sounds good